QModManager/API/SMLHelper/Utility/ExtBannedIdManager.cs

```diff
@@ -0,0 +1,141 @@
+﻿namespace QModManager.API.SMLHelper.Utility
+{
+    using QModManager.Utility;
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Reflection;
+
+    /// <summary>
+    /// This class is tasked with checking external txt files for banned IDs that are not to be isssued when patching in new enum entries.
+    /// </summary>
+    internal static class ExtBannedIdManager
+    {
+        private static bool IsInitialized = false;
+
+        private static readonly Dictionary<string, List<int>> BannedIdDictionary = new Dictionary<string, List<int>>();
+
+        private static readonly string BannedIdDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "RestrictedIDs");
```

Comment:
All path calls will need to be changed after the merge. I will handle this in #81, since it will also have path changes.